module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -188,46 +191,42 @@ namespace module::behaviour::strategy {
            With<Ball>,
            Single>()
             .then([this](const Sensors& sensors,
-                         const GameState& gameState,
+                         const GameState& game_state,
                          const Phase& phase,
-                         const FieldDescription& fieldDescription,
+                         const FieldDescription& field_description,
                          const Field& field,
                          const Ball& ball) {
                 try {
-
-                    // Force penalty shootout mode if set in config
-                    auto mode = cfg_.forcePenaltyShootout ? GameMode::PENALTY_SHOOTOUT : gameState.data.mode.value;
-
-                    kickType = KickType::SCRIPTED;  // Use the scripted kick
-
-                    Behaviour::State previousState = currentState;  // Update previous state
-
-                    // If force playing, just run the play functions based on mode
-                    if (cfg_.forcePlaying) {
-                        if (mode == GameMode::PENALTY_SHOOTOUT) {
-                            penaltyShootoutPlaying(field, ball);
-                        }
-                        else {
-                            normalPlaying(field, ball, fieldDescription);
-                        }
-                    }
                     // If we're picked up, stand still
-                    else if (pickedUp(sensors)) {
+                    if (picked_up(sensors)) {
                         // TODO(BehaviourTeam): stand, no moving
-                        standStill();
+                        stand_still();
                         currentState = Behaviour::State::PICKED_UP;
                     }
-
-                    // Switch gamemode statemachine based on GameController state
-                    switch (mode) {
-                        case GameMode::PENALTY_SHOOTOUT: penaltyShootout(phase, fieldDescription, field, ball); break;
-                        // We handle NORMAL and OVERTIME the same at the moment because we don't have any special
-                        // behaviour for overtime.
-                        case GameMode::NORMAL:
-                        case GameMode::OVERTIME: normal(gameState, phase, fieldDescription, field, ball); break;
-                        default: log<NUClear::WARN>("Game mode unknown.");
+                    else {
+                        // Overide SoccerStrategy and force normal playing behaviour
+                        if (cfg.force_playing) {
+                            normal_playing();
+                        }
+                        else {
+                            // Switch gamemode statemachine based on GameController state
+                            auto mode =
+                                cfg.force_penalty_shootout ? GameMode::PENALTY_SHOOTOUT : game_state.data.mode.value;
```

Comment:
This feels a little weird to be coded so differently to force playing. maybe make an if else statement above?